docs/rst/installation/linux_binary_installation.rst

```diff
@@ -3,12 +3,55 @@
 Linux binary installation
 =========================
 
-Debian packages for Vulcanexus Galactic Gamble are currently available for Ubuntu Focal.
-Please, start following the `Installation Guide for ROS 2 Galactic Geochelone <https://docs.ros.org/en/galactic/Installation/Ubuntu-Install-Debians.html>`_ setting the locale to support ``UTF-8`` (if necessary) and the ROS 2 apt repository.
-Instead of proceeding with installing the ROS 2 packages, follow the steps below.
+Debian packages for Vulcanexus Galactic Gamble are currently available for Ubuntu Focal. Since Vulcanexus is a ROS 2 overlay, certain ROS 2 prerequisites need to be met before installing.
```

Comment:
I am not sure if Operations would like to refer to Vulcanexus as a `ROS 2 overlay`. I think the official term is that Vulcanexus is a `ROS 2 all-in-one tool set`.